src/CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+set(definitions "UNDERSCORE" "NFILES=32" "MAXCD=250" "MAXNC=600" "MXNAF=3")
+add_compile_definitions(definitions)
+
+if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
+  set (FREAL8 "-r8")
+  set (FINT8 "-i8")
+  set (TRACEBACK "-traceback")
+  set (MMMED "-mcmodel=medium")
+  set (IntelComp TRUE)
+elseif(CMAKE_C_COMPILER_ID MATCHES "^(GNU|Clang|AppleClang)$")
+  set (FREAL8 "-fdefault-double-8" "-fdefault-real-8")
+  set (FINT8 "-fdefault-integer-8")
+  set (TRACEBACK "-fbacktrace")
+  set (MMMED "")
+endif()
+
+
+set(fortran_8_flags ${FREAL8} ${FINT8} ${TRACEBACK})
+set(shared_flags  ${DEBINFO} ${FOPT3} ${AVX2})
+set(fortran_8_DA_flags ${FREAL8} ${FINT8} ${TRACEBACK})
+set(fortran_d_DA_flags ${FREAL8} ${TRACEBACK})
+set(c_SS_flags ${MMMED} ${SHARED})
+
+
+set(c_DA_allocation_def "DYNAMIC_ALLOCATION")
+set(c_nonDA_allocation_def "STATIC_ALLOCATION")
+
+set(c_8_DA_definitions "F77_INTSIZE_8")
```

Comment:
the values of  `c_8_DA_definitions` is the same as `c_8_definitions`. i.e. it does not distinguish between `DA` and `nonDA`.  This can be reduced to `c_8_definitions`